alibi/explainers/tests/test_simiarlity/test_backends.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import torch
+import numpy as np
+import tensorflow as tf
+
+from alibi.explainers.similarity.backends.tensorflow.base import _TensorFlowBackend
+from alibi.explainers.similarity.backends.pytorch.base import _TorchBackend
+
+# ensure deterministic results
+tf.random.set_seed(0)
+np.random.seed(0)
+torch.manual_seed(0)
```

Comment:
I would add those internal to each test. maybe try considering multiple seeds in each test.